pkg/workspace/inference/preset-inferences.go

```diff
@@ -210,15 +251,24 @@ func CreatePresetInference(ctx context.Context, workspaceObj *v1beta1.Workspace,
 		},
 	})
 
+	// PYTORCH_CUDA_ALLOC_CONF with expandable_segments:True doesn't work with
+	// vLLM's custom_all_reduce - https://github.com/vllm-project/vllm/issues/9046
+	if runtimeName != pkgmodel.RuntimeNameVLLM {
+		envVars = append(envVars, corev1.EnvVar{
```

Comment:
The initial idea adding this env was to reduce mem fragments. If I understand correctly, will this break multi node inference?

Reply:
I have been encountering issues in https://github.com/vllm-project/vllm/issues/9046 with this environment variable. Let me check if it's an issue with existing preset models

Reply:
I think it only happens when using vLLM with tensor parallelism > 1. Could you give it a try as well?
![image](https://github.com/user-attachments/assets/b8f48c92-f259-4e73-a441-99c3ba017ed6)

Reply:
Moved the discussion to https://github.com/kaito-project/kaito/issues/1077

Reply:
Reverted the change since it appears to not work on certain GPUs only